wagtail/wagtailcore/fields.py

```diff
@@ -35,8 +45,13 @@ def value_from_datadict(self, data, files, name):
 
 
 class RichTextField(models.TextField):
+
+    def __init__(self, hallo_config=None, **kwargs):
```

Comment:
I've been following the discussion over here about making rich text areas pluggable: https://github.com/torchbox/wagtail/pull/1521#issuecomment-157022318

Do we want to create a common naming convention for configuring various RichText widgets? It would make sense if the property was called `config` or `editor_config` or similar: 

```python
widget=RichTextArea(config={
            'halloheadings': {...},
})

widget=TrixTextArea(config={
            'formats': {...},
})

widget=TinyMiceTextArea(config={
            'plugins' : 'advlist autolink link image lists charmap print preview',
})
```